apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.Tests/SpannerTransactionOptionsTests.cs

```diff
@@ -110,7 +110,7 @@ public void CommitPriority_Effective(Priority? priority, TransactionMode mode, P
     [InlineData(null, TransactionMode.ReadOnly, "")]
     [InlineData(null, TransactionMode.ReadWrite, "")]
     [InlineData("tag", TransactionMode.ReadOnly, "")] // Ignored for read-only
-    [InlineData("tag", TransactionMode.ReadWrite, "")]
+    [InlineData("tag", TransactionMode.ReadWrite, "tag")]
```

Comment:
I think this is effectively the fix for my comments in the previous commit - move the changes in this file to the previous commit?